packages/cli/test/integration.js

```diff
@@ -378,6 +392,91 @@ test('login', async t => {
   t.is(auth.token, token);
 });
 
+test('default command should deploy directory', async t => {
+  const projectDir = fixture('deploy-default-with-sub-directory');
+  const target = 'output';
+
+  await vcLink(t, path.join(projectDir, target));
+
+  const { exitCode, stderr, stdout } = await execa(
+    binaryPath,
+    [
+      // omit the default "deploy" command
+      target,
+      ...defaultArgs,
+    ],
+    {
+      cwd: projectDir,
+    }
+  );
+
+  t.is(exitCode, 0, formatOutput({ stdout, stderr }));
+  t.regex(stdout, /https:\/\/output-.+\.vercel\.app/);
+  t.regex(
+    stderr || '',
+    /calling `vc` without a command or the `--cwd` option is deprecated, use `vc --cwd directory` or `vc deploy` instead/
+  );
+});
+
+test('default command should warn when deploying with conflicting subdirectory', async t => {
+  const projectDir = fixture('deploy-default-with-conflicting-sub-directory');
+  const target = 'list'; // command that conflicts with a sub directory
+
+  await vcLink(t, projectDir);
+
+  const { exitCode, stderr, stdout } = await execa(
+    binaryPath,
+    [
+      // omit the default "deploy" command
+      target,
+      ...defaultArgs,
+    ],
+    {
+      cwd: projectDir,
+    }
+  );
+
+  t.is(exitCode, 0, formatOutput({ stdout, stderr }));
+  t.regex(
+    stderr || '',
+    /Did you mean to deploy the subdirectory "list"\? Use `vc --cwd list` instead./
+  );
+
+  const listHeader = /project +latest deployment +state +age +username/;
+  t.regex(stdout || '', listHeader); // ensure `list` command still ran
+});
+
+test('default command should work with --cwd option', async t => {
```

Comment:
I believe I had this test set up to link properly at one point, but I can't get it. Can someone who better understands how linking works help be get this test set up?

You can see the test failure here: https://github.com/vercel/vercel/runs/5573632410?check_suite_focus=true